functionary/ui/forms/scheduled_task.py

```diff
@@ -52,11 +74,20 @@ class Meta:
             "parameters",
         ]
 
-    def __init__(self, environment: Environment = None, *args, **kwargs):
+    def __init__(
+        self,
+        environment: Environment = None,
+        existing_errors=None,
+        *args,
+        **kwargs,
+    ):
         super().__init__(*args, **kwargs)
         self._setup_field_choices(kwargs.get("instance") is not None)
         self._update_function_queryset(environment)
         self._setup_field_classes()
+        self.existing_errors = (
```

Comment:
I'm not in love with this `existing_errors` things.  It seems like it would be more straightforward to just handle the case where  `task_parameter_form.is_valid()` returns `False` in the `ScheduledTaskCreateView`.  Right now it does:

```python
if task_parameter_form.is_valid():
   pass
```

and then proceeds to still construct the `ScheduledTaskForm` and all that.  I think that's why you end up with both the form field error and non-form-field errors.  You could instead introduce handling and do something like:

```python
if not task_parameter_form.is_valid():
    context = {
        "form": self.get_form(),
        "task_parameter_form": task_parameter_form,
    }
    return render(
        request, "forms/scheduled_task/scheduled_task_edit.html", context
    )
```

That creates some minor duplication with what gets returned at the end of that method, so there's probably some room for refactor there to reduce that, but I feel like doing something like that to handle the `TaskParameterForm` errors before generating moving on to do the `ScheduledTaskForm` handling would be cleaner.